fixham/pkg/goyek/hashrelease.go

```diff
@@ -0,0 +1,117 @@
+package goyek
+
+import (
+	_goyek "github.com/goyek/goyek/v2"
```

Comment:
this should probably be `goyekv2` instead of `_goyek` 

see my other comment re: the use of underscores in Golang variable and package names.